frontend/ui/wikipedia.lua

```diff
@@ -143,23 +272,40 @@ function Wikipedia:wikintro(text, lang)
     if result then
         local query = result.query
         if query then
+            local show_image = G_reader_settings:nilOrTrue("wikipedia_show_image")
+            -- Scale wikipedia normalized (we hope) thumbnail by 2 (adjusted
+            -- to screen size/dpi) for intros (and x8 more for highres image)
+            local image_size_factor = Screen:scaleBySize(200)/100.0
+            if show_image then
+                for pageid, page in pairs(query.pages) do
+                    self:addImages(page, lang, false, image_size_factor, 8)
+                end
+            end
             return query.pages
         end
     end
 end
 
 -- get full content of a wiki page
-function Wikipedia:wikifull(text, lang)
-    local result = self:loadPage(text, lang, WIKIPEDIA_FULL, true)
+function Wikipedia:wikifull(wiki_title, lang)
```

Comment:
I guess I missed these previously but functions are `lowerCamelCase` per the style guide. So wikiFull, wikiImages, etc.